src/key/crypto.ts

```diff
@@ -0,0 +1,86 @@
+import * as _ from "lodash";
+import { toArray, toHex } from "../utility";
+
+export type Ed25519Signature = string;
+
+export class Crypto {
+    public static async init() {
+        const sodium = require("libsodium-wrappers-sumo");
+        await sodium.ready;
+        const crypto = new Crypto(sodium);
+
+        return crypto;
+    }
+
+    public key: Key;
+    public signature: Signature;
+    private constructor(sodium: any) {
+        this.key = new Key(sodium);
+        this.signature = new Signature(sodium);
+    }
+}
+
+class Key {
```

Comment:
We don't have to write all the classes in this file. What do you think of this change? @majecty

Reply:
I agree with you. Let's split the classes.

Reply:
Can I combine class `Crypto` and class `Key` into key.ts file, and move class `Signature` to signature.ts file? It means two files, key.ts and signature.ts.

Reply:
I think it's better to separate into three files, `key.ts`, `ed25519.ts` and `crypto.ts`. I modified it.